org-cyf-itd/content/blocks/step-1/instructions/index.md

```diff
@@ -9,14 +9,50 @@ time= 60
+
+You don’t need to understand the technical terms in these two pieces of text to complete the task.
+
+###### Jane’s summary:
+Jane has recently completed the part time Super Coders Software Development programme. During this time, she was also working full-time and caring for her two children. The course teaches full stack web development and general programming concepts. It teaches HTML, CSS, JavaScript, Node.js, Express, React and PostgreSQL. Jane is very passionate about programming. She taught herself the Python programming language before starting the course.
+
+Her portfolio includes many full stack applications.This includes projects she has built with other students, as well as some solo projects. She’s especially proud of a project called Pin the Mood. This project stores book recommendations until the user has time to read the books. The project used React, Express, Node.js and PostgresSQL. The project team was an agile team made up of two trainees, a scrum master and a tech lead.
+
+After leaving school seven years ago, she joined Lawyers 4U as a call handler. She has developed excellent interpersonal skills and has gained experience working under pressure.
+
+###### The Job Description:
+Software for Lawyers Ltd makes case management software. Their software helps lawyers manage caseloads and track time spent. The development team is small and close knit. They are looking to add a Junior Software Developer to their team.
+
+**Requirements**
+- A university degree in a relevant subject or software development course
+- Experience with Node.js, Express and React
+- A strong portfolio with solo and group projects
+- Excellent interpersonal skills
+- Experience developing software in an agile team
+- Passionate about programming
+
+**Nice to Haves**
+- Experience in the legal sector
+- Some knowledge of Python
+- Experience working in an office-based environment
+
+##### Steps
+1. Go to https://chatgpt.com
+2. Signup for an account
+3. Ask the AI to help you write a cover letter. Give it the description of Jane and the job description provided. It may take some experimentation to get the right prompt. That’s OK - getting AI to do what you want is all about trial and error.
+4. Read and reflect on the cover letter. How has the AI demonstrated that Jane has the right skills for the job? Has the AI added anything that wasn’t in the description of Jane? Could you do better?
+5. Create a document with the text of the cover letter you generated
+6. Save the document. Please ensure that your name is in the file name. It can just be your given name or your family name if you prefer.
+7. Upload the cover letter to this site
+8. Add a comment with at least one comment (positive or negative) on the AI’s work
```

Comment:
```suggestion
5. Think about your own future job hunt. What have you learned that will help you apply for jobs in the future? Will you use ChatGPT to help you generate a cover letter? If so, how will you make sure it's a good cover letter? If not, why not?
6. Create a document with the text of the cover letter you generated
7. Save the document. Please ensure that your name is in the file name. It can just be your given name or your family name if you prefer.
8. Upload the cover letter to this site
9. Add a comment with at least three comments (positive or negative) on the AI’s work
```